Change documentation for Continuous Testing (#15736)
DataDog/documentation: content/en/continuous_testing/cicd_integrations/configuration.md

: **Type**: string<br>
New start URL to provide to the HTTP or browser test.

`startUrlSubstitutionRegex`
: **Type**: string<br>

> Regex to modify the start URL of the test (browser and HTTP tests only), whether it was given by the original test or by the configuration override startUrl. If the URL contains variables, this regex will be applied after the interpolation of the variables.

Reviewer: ```suggestion
Regex to modify the start URL of the test (browser and HTTP tests only), whether it was given by the original test or by the configuration override `startURL`. If the URL contains variables, this regex is applied after the interpolation of the variables.
```

The execution rule associated with the test is the most restrictive one in the configuration file. The options range from most to least restrictive: `skipped`, `non_blocking`, and `blocking`. For example, if your test is configured as `skipped` in the UI but `blocking` in the configuration file, it is `skipped` when your test runs.

> #### Customizing your start URL

Reviewer: ```suggestion
#### Customize your start URL
```

> You can override the start URL for your tests through the `startURL` configuration option. For example, your test starting URL is `shopist.io`, but you want to test your staging environment at `staging.shopist.io`, you would pass the option as `starURL: "staging.shopist.io"`.

Reviewer: ```suggestion
You can override the start URL for your tests through the `startURL` configuration option. For example, assuming your test starting URL is `shopist.io`, and you want to test your staging environment at `staging.shopist.io`, pass the option as `startURL: "staging.shopist.io"`.
```

> If you want to customize this start URL further (or only part of the URL), you can use the `startUrlSubstitutionRegex` configuration option. The format is `s/your_regex/your_substitution/modifiers` and follow Javascript regex syntax. For example, s/(https://www.)(.*)/$1extra-$2/ to transform https://www.example.com into https://www.extra-example.com

Reviewer: ```suggestion
If you want to customize this start URL further (or only part of the URL), you can use the `startUrlSubstitutionRegex` configuration option. The format is `s/your_regex/your_substitution/modifiers` and follows JavaScript regex syntax. For example, `s/(https://www.)(.*)/$1extra-$2/` transforms  `https://www.example.com` into `https://www.extra-example.com`.
```
